pynucastro/networks/rate_collection.py

```diff
@@ -569,6 +571,247 @@ def evaluate_rates(self, rho, T, composition):
 
         return rvals
 
+    def _evaluate_comp_NSE(self, u, rho, T, composition, ye=None):
+        """ A helper equationt that finds the mass fraction of each nuclide 
+        in NSE state, u[0] is mu_p^C while u[1] is bar{mu_n}"""
+        
+        m_u = constants.value("unified atomic mass unit")*1.0e3  # atomic unit mass in g
+        k = constants.value("Boltzmann constant")*1.0e7          # boltzmann in erg/K
+        h = constants.value("Planck constant")*1.0e7             # in cgs
+        e = 4.8032e-10                                           # electron charge in cgs
+
+        A_1 = -0.9052
+        A_2 = 0.6322
+        A_3 = -0.5*np.sqrt(3.0)-A_1/np.sqrt(A_2)
```

Comment:
what are these numbers?  can you add a comment saying what they mean?

Reply:
yes, I'll add more comments later tonight!